src/message.cpp

```diff
@@ -279,6 +279,12 @@ void term(const char *fmt, ...)
   exit(1);
 }
 
+void warn_flush(void)
```

Comment:
void here is also useless (it is the default in C++ unlike in C)

Reply:
I'm more from the old sort, I like the clarity.
I can change it without problems.